### ISSUE TITLE
Vertical screen overflow

### ISSUE THREAD

**Issue by sander-spruit** [trvswgnr/bs5-lightbox#2]
Although the gallery works great, vertical images overflow the screen size and create a scroll, rather than being scaled back to fit the screen.

Edit: temporary fix;
```
.carousel-item img {
    width: auto !important;
    max-height: 90vh;
    margin: 0 auto;
}
```

**Comment by trvswgnr** [comment]
Thank you for bringing this to my attention. I'll take a look and patch in the next release.

**Comment by Erulezz** [comment]
Same issue here:
<img width="374" alt="Schermafbeelding 2021-11-20 om 14 34 47" src="https://user-images.githubusercontent.com/6772197/142728329-8544d47e-d923-4e8e-aca0-6d24cf2635b8.png">

All default settings, very big images (4000x3000) and I have scrollbars in all directions.

**Comment by trvswgnr** [comment]
I see the issue but I'm not sure what I want the default behavior to be. Will implement a fix soon.

**Comment by shriharshmishra** [comment]
It would be great to have this feature.

**Comment by trvswgnr** [comment]
I believe this has been fixed in v1.7.11.  Let me know if you’re still having this issue.
